templates/05-token-contract/setup.js

```diff
@@ -4,8 +4,8 @@
 require.config({
   paths: {
     "binaryen": "https://cdn.jsdelivr.net/gh/AssemblyScript/binaryen.js@84.0.0-nightly.20190522/index",
-    "assemblyscript": "https://cdn.jsdelivr.net/gh/nearprotocol/assemblyscript@d8c4f1de18614c966f19b41f89ace2ef5cd876c5/dist/assemblyscript",
```

Comment:
@willemneal any reason to migrate from jsdelivr to githack?

Reply:
Also generally we want to pin specific version of compiler, as depending on latest compiler will break old fiddles more frequently.